hyperparameters.md

```diff
@@ -34,6 +34,6 @@ ServingRateLimit: u64 = 250; // [250 @ 7,166]
 Burn: u64 = 100_000_000_000; // 100 tao [100 tao @ 26310]
 MinBurn: u64 = 10_000_000_000; // 10 tao [10 tao @ 26310]
 MaxBurn: u64 = 21_000_000_000_000_000; // 21M tao [21M tao @ 26310]
-TxRateLimit: u64 = 1000;
+TxRateLimit: u64 = 50;
```

Comment:
```suggestion
TxRateLimit: u64 = 1;
```
I suggest lower to 1 or 2 until a different mechanism is in-place for staking/unstaking